src/data/config.json

```diff
@@ -162,6 +162,11 @@
             "methods": ["signTransaction"],
             "min": ["1.10.1", "2.4.0"],
             "comment": [""]
+        },
+        {
+            "methods": ["ethereumSignTypedData"],
+            "min": ["0", "2.4.2"],
+            "comment": ["EIP-712 typed signing support added in 2.4.2"]
```

Comment:
it should be 2.4.3

Reply:
Good catch!

I've updated it. I also made sure sure to stick `(Trezor T only)` into the CHANGELOG.md entry.